gno.land/pkg/sdk/vm/convert.go

```diff
@@ -153,6 +140,34 @@ func convertArgToGno(arg string, argT gno.Type) (tv gno.TypedValue) {
 			}
 			tv.SetUint64(u64)
 			return
+		case gno.Float32Type:
+			assertCharNotPlus(arg[0])
+			dec, _, err := apd.NewFromString(arg)
+			if err != nil {
+				panic(fmt.Sprintf("error parsing float32 %q: %v", arg, err))
```

Comment:
```suggestion
				panic(fmt.Sprintf(
					"error parsing float32 %q: %v",
					arg, err))
```

I don't have any hard preferences here. But i think its a good idea to follow the existing formatting style :)

Same with other `panic`'s

Reply:
I see what you mean but I don't really like the existing style.  In my opinion you can have it the way I've done it, so long that the line length isn't overly long, or you can make it be formatted vertically, though the existing style isn't the ideal form of vertical formatting.  I think would be something like this to avoid multiple parentheses on the same line and clearly distinguish each of the arguments provided to a function:
```
panic(
    fmt.Sprintf(
        "error parsing float32 %q: %v",
        arg,
        err,
    ),
)
```
I think the purpose of this style of formatting is to increase the readability for lines of code that have lots of args, calls, literals, etc.  This is not the case in this scenario so I'm not sure it needs this and the one line approach is fine -- it's a panic that takes a string built using `fmt.Sprintf` -- nothing too complicated going on there.

I can also change it to be consistent with the others. I just wanted to explain my reasoning here.